std/os/file.zig

```diff
@@ -77,7 +79,10 @@ pub const File = struct {
     /// Call close to clean up.
     pub fn openWriteMode(path: []const u8, file_mode: Mode) OpenError!File {
         if (is_posix) {
-            const flags = posix.O_LARGEFILE | posix.O_WRONLY | posix.O_CREAT | posix.O_CLOEXEC | posix.O_TRUNC;
+            var flags: u32 = posix.O_WRONLY | posix.O_CREAT | posix.O_CLOEXEC | posix.O_TRUNC;
```

Comment:
what you have here is fine, but if you do it like this, you can keep `flags` as a comptime-known value:

```zig
const largefile = if (builtin.os == builtin.Os.linux) posix.O_LARGEFILE else 0;
const flags = posix.O_WRONLY | posix.O_CREAT | posix.O_CLOEXEC | posix.O_TRUNC | largefile;
```

Reply:
Good point, thanks.